### DISCUSSION TITLE
My Octolamp

### DISCUSSION THREAD

**Discussion by Likqez** [martinwoodward/octolamp#22] [Show and tell]
Hi, i used the Prusa i3 MK3S+ to print.

I only had IP65 LED Strips availbe and thus fitting it all inside the the small parts, especially inside the arm. In general a solid build.

## Some pics:

![20230606_230537](https://github.com/martinwoodward/octolamp/assets/41593553/51bc8c4d-e330-4936-96b4-816fe9f728cf)

![20230526_223610](https://github.com/martinwoodward/octolamp/assets/41593553/a87e0eb3-1da9-4a83-9c82-66770a0fd99c)

![image](https://github.com/martinwoodward/octolamp/assets/41593553/b078d80c-ad5d-477b-9066-d7f66229214f)

**Comment by martinwoodward** [comment]
Great build, very neat work with the LED routing, that tends to be the trickiest part.

**Reply by Likqez** [reply]
Thank you, but not quite. Now after 3 days some sections are coming loose again. I might have put too much stress on the corners. Hopefully extra double sided tape will do a better job than the strips glue itself.

**Reply by martinwoodward** [reply]
Oh no!  Maybe a few strategically placed drops of CA glue could help or even hot melt glue.  Good luck!
